src/main/java/net/dv8tion/jda/webhook/WebhookCluster.java

```diff
@@ -745,12 +748,20 @@ public WebhookCluster removeWebhooks(Collection<WebhookClient> clients)
     {
         Checks.notNull(file, "File");
         Checks.check(file.length() <= Message.MAX_FILE_SIZE, "Provided File exceeds the maximum size of 8MB!");
-        return broadcast(new WebhookMessageBuilder().setFile(file, fileName).build());
+        try
+        {
+            return broadcast(new WebhookMessageBuilder().addFile(fileName, file).build());
```

Comment:
I just realized this broadcast won't work because we drain the InputStream on first use